Oxide.Core/RemoteConsole/RemoteConsole.cs

```diff
@@ -201,6 +206,9 @@ public RconPlayer(IPlayer player)
                 Health = player.Health;
                 Ping = player.Ping;
                 ConnectedSeconds = 0; // Todo when support is added
+                VoiationLevel = 0; // Needed For Compat
```

Comment:
No need for Title Casing, also please clarify which game and such. Same applies to the other two lines as well.

Example:

// Needed for Rust compatibility